packages/jobs/lib/activities.ts

```diff
@@ -113,6 +121,13 @@ export async function scheduleAndRouteSync(args: ContinuousSyncArgs): Promise<bo
     let environmentId = nangoConnection?.environment_id;
     let syncJobId;
 
+    const isGloballyEnabled = await featureFlags.isEnabled('orchestrator:schedule', 'global', false);
+    const isEnvEnabled = await featureFlags.isEnabled('orchestrator:schedule', `${environmentId}`, false);
+    const isOrchestrator = isGloballyEnabled || isEnvEnabled;
+    if (isOrchestrator) {
+        return false;
+    }
+
```

Comment:
@khaliqgant Can you confirm those are the only two entry points for sync processing via Temporal? thank you

Reply:
Also is that ok to return `false` when sync is not processed via Temporal?

Reply:
👍 changing it to true. thank you